core/templates/dev/head/header_js_libs.html

```diff
@@ -2,27 +2,10 @@
   <script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
 <![endif]-->
 <!--[if gte IE 9]><!-->
-  <script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js">
+  <script src="/third_party/static/jquery-2.1.1/jquery.min.js">
   </script>
 <!--<![endif]-->
-<script>
-  window.jQuery || document.write(
-    '<script src="/third_party/static/jquery-2.1.1/jquery.min.js">\x3C/script>');
-</script>
-
-<script src="//ajax.googleapis.com/ajax/libs/jqueryui/1.10.3/jquery-ui.min.js"></script>
-<script>
-  window.jQuery.ui || document.write(
-    '<script src="/third_party/static/jqueryui-1.10.3/jquery-ui.min.js">\x3C/script>');
-</script>
-
-<script src="/third_party/static/bootstrap-3.3.4/js/bootstrap.min.js"></script>
-<script src="/third_party/static/angularjs-1.4.7/angular.js"></script>
-<script src="/third_party/static/angularjs-1.4.7/angular-aria.min.js"></script>
-<script src="/third_party/static/ui-bootstrap-0.13.4/ui-bootstrap-tpls-0.13.4.min.js"></script>
-<script src="/third_party/static/ng-joyride-0.1.11/ng-joyride.js"></script>
-<script src="/third_party/static/select2-3.5.1/select2.min.js"></script>
-
+<script  src="/third_party/static/angularjs-1.4.7/angular.min.js"></script>
```

Comment:
Done. I had an idea to add async attribute. so as they may  run asynchronously to reduce amount of time before the first content shows up. But actually looks like image slider depends much on angular to load. so it was trowing a warning.